lib/ansible/galaxy/data/community/README.md.j2

```diff
@@ -0,0 +1,59 @@
+# {{ namespace }}.{{ collection_name }} Collection
```

Comment:
Do we want to add a `Read me first` checklist here? 
* things to do
* files to possibly review & delete